internal/appsec/dyngo/instrumentation/httpinstr/http.go

```diff
@@ -112,12 +107,20 @@ func WrapHandler(handler http.Handler, span ddtrace.Span) http.Handler {
 				status = mw.Status()
 			}
 			op.Finish(HandlerOperationRes{Status: status})
+
+			if secEvent != nil {
+				remoteIP, _, err := net.SplitHostPort(r.RemoteAddr)
+				if err != nil {
+					remoteIP = r.RemoteAddr
+				}
+				setSpanTags(span, secEvent, remoteIP, headers)
```

Comment:
Improvement: the tags are set in the integration package which has access to everything we need to set the span tag, without having to pass them into the `HandlerOperationArgs` anymore.